template.tpl

```diff
@@ -767,104 +759,217 @@ ___TEMPLATE_PARAMETERS___
-            "macrosInSelect": true,
-            "selectItems": [
+            "type": "GROUP",
+            "name": "uaPaserOptions",
+            "displayName": "User Agent Options",
+            "groupStyle": "NO_ZIPPY",
+            "subParams": [
+              {
+                "type": "CHECKBOX",
+                "name": "osName",
+                "checkboxText": "Enrich us_name using client user agent enrichment",
+                "simpleValueType": true,
+                "defaultValue": true
+              },
+              {
+                "type": "CHECKBOX",
+                "name": "osVersion",
+                "checkboxText": "Enrich os_version using client user agent enrichment",
+                "simpleValueType": true,
+                "defaultValue": true
+              },
               {
-                "displayValue": "Page Loads (default)",
-                "value": "pageLoads"
+                "type": "CHECKBOX",
+                "name": "deviceManufacturer",
+                "checkboxText": "Enrich device_manufacturer using client user agent enrichment",
+                "simpleValueType": true,
+                "defaultValue": true
               },
               {
-                "displayValue": "Only with attribution changes",
-                "value": "attribution"
+                "type": "CHECKBOX",
+                "name": "deviceModel",
+                "checkboxText": "Enrich device_model using client user agent enrichment",
+                "simpleValueType": true,
+                "defaultValue": true
               }
             ],
             "enablingConditions": [
               {
-                "paramName": "attributionPageViewTracking",
-                "type": "EQUALS",
-                "paramValue": true
+                "paramName": "uaParser",
+                "paramValue": true,
+                "type": "EQUALS"
               }
-            ],
-            "displayName": "Page View trigger",
-            "defaultValue": "pageLoads",
-            "simpleValueType": true,
-            "name": "attributionPageViewTrackOn",
-            "type": "SELECT"
-          },
+            ]
+          }
+        ]
+      },
+      {
+        "help": "Check this box to enable default event tracking. \u003ca href\u003d\"https://www.docs.developers.amplitude.com/data/sdks/browser-2/#tracking-default-events\"\u003eRead more\u003c/a\u003e.",
+        "defaultValue": true,
```

Comment:
It makes sense to default to true to align with browser 2.0 configuration if we are creating a new template. But it's just an update on the existing template, it may send more events without customers being aware.

Reply:
Could this change be documented in the announcement email rather defaulting to false? Not having to enable DET would make setup easier for new customers.

Reply:
As we discussed, we will make the default behavior the same as det browser 2.0 and let the customer opt out.